index.html

```diff
@@ -36,6 +36,321 @@ <h5>LET’S CONNECT</h5>
              </ul>
             </div>
         </section>
+        <section class="Snapshoot">
+
+
+          <article class="article1">
+           <figure>
+            <img src="images/Snapshoot Portfolio.png" alt="Portfolio1">
+           </figure>
+
+
+              <div class="article1div">
+
+
+                  <h3 class="texttitle font1">Tonic</h3>
+
+
+                  <ul>
+
+
+                    <li class="texttitle font2">CANOPY</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">Back End Dev</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">2015</li>
+
+
+                  </ul>
+
+
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+
+                  <ul class="float_box1">
+
+
+                      <li class="background1"><a class="texttitle font4" href="#">html</a></li>
+                      
+                      <li class="background1"><a class="texttitle font4" href="#">css</a></li>
+                      
+                      <li class="background1"><a class="texttitle font4" href="#">javaScript</a></li>
+
+
+                  </ul>
+
+
+                  <button type="submit"><a class="font5 texttitle" href="#">See Project</a></button>
+
+
+              </div>
+
+
+          </article>
+
+
+          <article class="article2">
+
+
+              <figure>
+
+
+         <img src="images/Snapshoot Portfolio (1).png" alt="Portfolio2">
+
+
+              </figure>
+
+
+              <div class="article2div">
+
+
+                  <h3 class="texttitle font1">Multi-Post Stories</h3>
+
+
+                  <ul>
+
+
+                    <li class="texttitle font2">CANOPY</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">Back End Dev</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">2015</li>
```

Comment:
- Kindly, be sure that the bullet icons are visible 👍👌 and it is best to use style instead of image to display bullets👌👍